enterprise/tpa_pipeline.py

```diff
@@ -105,7 +105,7 @@ def get_consent_status_for_pipeline(pipeline):
 
 
 @partial
-def verify_data_sharing_consent(backend, user, **kwargs):
+def handle_enterprise_logistration(backend, user, **kwargs):
     """
     Verify that the data sharing consent state matches what's required.
 
```

Comment:
Let's update the docstring to reflect the correct format while we're here (Args, Returns)